Patch from gpg2win.
Got it from:
https://wald.intevation.org/plugins/scmsvn/viewcvs.php/*checkout*/trunk/patches/gpgme/01-gpg2.patch?rev=666&root=gpg4win

Change gpgme to use gpg2.exe by default.

--- gpgme/w32-util.c
+++ gpgme/w32-util.c
@@ -307,9 +307,9 @@
   if (!gpg_program)
     gpg_program = find_program_in_registry ("gpgProgram");
   if (!gpg_program)
-    gpg_program = find_program_in_inst_dir ("gpg.exe");
+    gpg_program = find_program_in_inst_dir ("gpg2.exe");
   if (!gpg_program)
-    gpg_program = find_program_at_standard_place ("GNU\\GnuPG\\gpg.exe");
+    gpg_program = find_program_at_standard_place ("GNU\\GnuPG\\gpg2.exe");
   UNLOCK (get_path_lock);
   return gpg_program;
 }
